helm/hpcc/templates/localroxie.yaml

```diff
@@ -68,7 +68,7 @@ spec:
         server: {{ $servername | quote }}
         accessDali: "yes"
         accessEsp: "yes"
-        helmVersion: 8.8.0-closedown0
+        {{- include "hpcc.addStandardLabels" (dict "root" $ "component" $servername "name" "roxie" "instance" $roxie.name) | indent 8 }}
```

Comment:
I don't think "component" should be user defined (or rather values.yaml defined), it should a known constant.
I think "roxie-server" is appropriate here  (same as in roxie.yaml)

Reply:
corrected: should be user defined -> shouldn't be user defined